coral-erm/coral
# Issue 599: Fix dashboards sums for resources with multiple subjects.

Background: issue #599 : Multiple subjects in dashbord show wrong sum

Test plan:

 - Find a resource with several subjects attached.
 - Check that the sum displayed in the payment amount column is wrongly
   multiplied by the number of attached subjects.
 - Apply the patch.
 - Check that the payment amount value is now correct, in both statistics
   and yearly costs dashboards, using filters or not.
